stdlib/src/builtin/math.mojo

```diff
@@ -250,3 +253,20 @@ fn round[T: Roundable](value: T) -> T:
         The rounded value of the object.
     """
     return value.__round__()
+
+
+@always_inline
+fn round[T: Roundable](value: T, ndigits: IntLiteral) -> T:
```

Comment:
See my other comment.

Reply:
Done in same commit: https://github.com/msaelices/mojo/commit/9de5acca6db1dad73f160a084687cdfaba9f5ed4